webapp/views.py

```diff
@@ -75,7 +75,28 @@ def _normalise_resources(self, resources):
     def _get_resources(self):
         topic = self.request.GET.get('topic')
         content = self.request.GET.get('content')
+        search = self.request.GET.get('search')
```

Comment:
I think this should be `?q=` and not `?search=`. It is the defacto standard for search across the web.

Reply:
Good spot @WillMoggridge

Reply:
Done